Makefile

```diff
@@ -85,7 +85,7 @@ docs/assets/js/swagger.yml: api/swagger.yml
 docs: docs/assets/js/swagger.yml
 
 docs-serve: ### Serve local docs
-	cd docs; bundle exec jekyll serve
+	cd docs; bundle exec jekyll serve --livereload
```

Comment:
Trying looking at it - but do you know what's the difference between this and the previous command (which seemed to update the docs as well)?

Reply:
`serve` requires you to reload the page to see updates; `--livereload` makes it happen automagically (the client auto-reloads when a change is made)